components/sandbox/index.js

```diff
@@ -141,12 +147,18 @@ export default class Sandbox extends Component {
 			body {
 				margin: 0;
 			}
+			
+			body.html {
+				width: 100%;
+			}
+
 			body.video,
 			body.video > div,
 			body.video > div > iframe {
 				width: 100%;
 				height: 100%;
```

Comment:
is it going to harm anyone to apply height: 100% in all cases? I'm using it on my branch based upon your work where everything has `{"type":"html"}`. It works for videos & audio & I've not had any issues.

Reply:
@Lewiscowles1986 Regarding changes done to Sandbox component - have you verified if it doesn't the `embed` block? This is because the embed block uses this component too. Check <a href="https://github.com/WordPress/gutenberg/blob/master/blocks/library/embed/index.js#L208">here</a>.

Reply:
I tested it with oEmbeds by typing a youtube URL into the shortcode block. Seemed to work

Reply:
tbf I changed `height: 100%;` to `min-height: 100%;`

Reply:
@Lewiscowles1986 I was referring to the embed block - not embeds in the shortcode block.

![screen shot 2018-02-24 at 11 54 34 pm](https://user-images.githubusercontent.com/1269602/36633567-2cda81ce-19be-11e8-9afc-3721eefb4cf0.png)

Reply:
Works in embed block still :wink: 
![screenshot capture - 2018-02-24 - 20-24-54](https://user-images.githubusercontent.com/2605791/36634727-ce5e1d58-19a0-11e8-9b3a-b92771553736.png)
![screenshot capture - 2018-02-24 - 20-24-45](https://user-images.githubusercontent.com/2605791/36634728-ce7999de-19a0-11e8-8cb6-91fa50e26cde.png)

I have found a problem with the embed block though. It seems intended to only render the block once without an editor like shortcode block has
[gutenberg.zip](https://github.com/WordPress/gutenberg/files/1755198/gutenberg.zip)

I can share my commit once I push to github